sphinx_pyscript/__init__.py

```diff
@@ -10,23 +10,29 @@
 from docutils.parsers.rst import directives
 from sphinx.application import Sphinx
 from sphinx.util.docutils import SphinxDirective
+from sphinx.util.fileutil import copy_asset_file
 from sphinx.util.logging import getLogger
 
+DEFAULT_VERSION = "2024.4.2"
+
 
 def setup(app: Sphinx):
     """Setup the extension"""
     app.add_config_value(
-        "pyscript_js", "https://pyscript.net/releases/2022.12.1/pyscript.js", "env"
+        "pyscript_js", f"https://pyscript.net/releases/{DEFAULT_VERSION}/core.js", "env"
     )
     app.add_config_value(
-        "pyscript_css", "https://pyscript.net/releases/2022.12.1/pyscript.css", "env"
+        "pyscript_css",
+        f"https://pyscript.net/releases/{DEFAULT_VERSION}/core.css",
+        "env",
     )
     app.add_directive("py-config", PyConfig)
     app.add_directive("py-script", PyScript)
-    app.add_directive("py-repl", PyRepl)
+    app.add_directive("py-editor", PyEditor)
     app.add_directive("py-terminal", PyTerminal)
     app.connect("doctree-read", doctree_read)
     app.connect("html-page-context", add_html_context)
+    app.connect("build-finished", copy_asset_files)
```

Comment:
was there a reason to have this in `build-finished`?
I think `env-updated` is the better place (at least I know I use this in another package)

Reply:
When I tried `env-updated`, mini-coi.js doesn't end up getting copied from the extension package to the output static directory successfully. I also tried using `doctree-resolved`, but it seems to really does need to be as late as `build-finished` to successfully copy over.

Reply:
I literally just updated some code a few days ago, where we use `env-updated` to copy JS:  https://github.com/useblocks/sphinx-needs/blob/master/sphinx_needs/environment.py, so it definitely can work 😄.
I can have a look at the code here, but it might be a day or two

Reply:
Thank you! Let me take another swing at it today, and see if I can match what you're doing in `sphinx_needs`.

Reply:
@chrisjsewell Success! And I got to learn a little more about Sphinx core events along the way. 

I also fixed a couple things in things in the tests that I missed the first time around, so those now pass as well.